src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,42 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMUM_LENGTH = 6;
+    private static final int MINIMUM_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (storageDao.get(user.getLogin()) == null) {
```

Comment:
Bad approach to use if and a lot of inner if conditions
Lets make it better 
check user == null
create separate methods:
validateLogin -> storageDao.get(user.getLogin()) != null and login == null and user.getPassword().length() < MINIMUM_LENGTH
validatePassword -> null and length
validateAge -> null and length